Test/Utils/ContrailAPI_New/Models/SecurityGroupModel.ps1

```diff
@@ -0,0 +1,55 @@
+# Those are just informative to show dependencies
+#include "PolicyRuleModel.ps1"
+
+class SecurityGroup : BaseResourceModel {
+    [String] $Name
+    [String] $ProjectName
+    [String] $DomainName = 'default-domain'
+    [PolicyRule[]] $PolicyRules = @()
+
+    SecurityGroup([String] $Name, [String] $ProjectName) {
+        $this.Name = $Name
+        $this.ProjectName = $ProjectName
+    }
+
+    [String[]] GetFqName() {
+        return @($this.DomainName, $this.ProjectName, $this.Name)
+    }
+
+    static [SecurityGroup] new_Default([String] $ProjectName) {
+        $group = [SecurityGroup]::new('default', $ProjectName)
```

Comment:
Please supply a comment explaining those calls (about allowing all incoming and outgoing packets)

Reply:
```suggestion
        $group = [SecurityGroup]::new('default', $ProjectName)
        # These two, created below, policy rules
        # allow network traffic to and from security group.
```

Reply:
@michal-clapinski Please look at my suggestions about the comments before I commit them (not to trigger useless jobs at CI).